zetta_utils/mazepa_layer_processing/common/volumetric_apply_flow.py

```diff
@@ -414,10 +414,16 @@ def flow(  # pylint:disable=too-many-branches, too-many-statements
             logger.info(f"Submitting {len(tasks)} processing tasks from operation {self.op}.")
             yield tasks
             yield mazepa.Dependency()
+            if not self.max_reduction_chunk_size_final >= dst.backend.get_chunk_size(
+                self.dst_resolution
+            ):
+                copy_chunk_size = dst.backend.get_chunk_size(self.dst_resolution)
+            else:
+                copy_chunk_size = self.max_reduction_chunk_size_final
```

Comment:
No - the behaviour is a little different when they're not comparable.

Reply:
Oof, that feels like relying on Vec3D magic too much. `not <=` will be pretty confusing for somebody to read

Reply:
They can be not comparable because one of them is `None`, or are there other scenarios?

Reply:
Oh if it's one of them is not strictly larger hmmm

Reply:
Correct - (1, 2, 3) and (1, 0, 4) for example. Max defaults to the first one, so we could use it and it would work, but that's even more confusing. Maybe the move is to implement dimwisemax or something like that.

Reply:
Sg, I will make an issue and merge this for now